src/ert/_c_wrappers/job_queue/workflow_job.py

```diff
@@ -48,6 +52,14 @@ class WorkflowJob:
 
     def __post_init__(self):
         self.__running = False
+        self._ert_script: Optional[type] = None
+        if self.script is not None and self.internal:
+            try:
+                self._ert_script = ErtScript.loadScriptFromFile(self.script)  # type: ignore
+            except Exception as err:
```

Comment:
Can you add the pylint exception here right away (and explain why), then whenever that pylint issue on bare exceptions, it will not be touched :)

Reply:
Done